Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Call.hs

```diff
@@ -50,13 +50,22 @@ directCallStatusCallback callSid dialCallStatus recordingUrl_ callDuration = do
   runTransaction $ QCallStatus.updateCallStatus callStatus.id (exotelStatusToInterfaceStatus dialCallStatus) (fromMaybe 0 callDuration) recordingUrl
   return Ack
 
-getCustomerMobileNumber :: (CacheFlow m r, EsqDBFlow m r, EsqDBReplicaFlow m r, EncFlow m r) => Text -> Text -> Text -> ExotelCallStatus -> m GetCustomerMobileNumberResp
-getCustomerMobileNumber callSid callFrom_ callTo_ callStatus = do
+getCustomerMobileNumber :: (CacheFlow m r, EsqDBFlow m r, EsqDBReplicaFlow m r, EncFlow m r) => Text -> Text -> Text -> Text -> ExotelCallStatus -> m GetCustomerMobileNumberResp
+getCustomerMobileNumber callSid callFrom_ callTo_ digits callStatus = do
```

Comment:
name it `dtmfNumber` instead of digits, and check if 0 is there at start if yes then remove it.